.cirrus.yml

```
@@ -133,49 +133,97 @@ windows_task:
+        container_img: ghcr.io/cirruslabs/macos-ventura-xcode:latest
+        prepare: echo libtool autoconf automake pkg-config nghttp2 | xargs -Ix -n1 echo brew '"x"' > /tmp/Brewfile
+        configure: --enable-warnings --enable-werror --enable-debug --enable-maintainer-mode --disable-dict --disable-file --disable-ftp --disable-gopher --disable-imap --disable-ldap --disable-pop3 --disable-rtmp --disable-rtsp --disable-scp --disable-sftp --disable-smb --disable-smtp --disable-telnet --disable-tftp --disable-unix-sockets --disable-shared --without-brotli --without-gssapi --without-libidn2 --without-libpsl --without-librtmp --without-libssh2 --without-nghttp2 --without-ntlm-auth --without-ssl --without-zlib --enable-websockets
+
+    - name: macOS arm64 SecureTransport http2
+      env:
+        container_img: ghcr.io/cirruslabs/macos-ventura-xcode:latest
+        prepare: echo libtool autoconf automake pkg-config nghttp2 | xargs -Ix -n1 echo brew '"x"' > /tmp/Brewfile
+        configure: --enable-warnings --enable-werror --enable-debug --with-secure-transport --enable-websockets
+
+    - name: macOS arm64 OpenSSL http2
+      env:
+        container_img: ghcr.io/cirruslabs/macos-ventura-xcode:latest
+        prepare: echo libtool autoconf automake pkg-config nghttp2 openssl | xargs -Ix -n1 echo brew '"x"' > /tmp/Brewfile
+        configure: --enable-warnings --enable-werror --enable-debug --with-openssl=$(brew --prefix openssl) --enable-websockets
+
+    - name: macOS arm64 LibreSSL http2
+      env:
+        container_img: ghcr.io/cirruslabs/macos-ventura-xcode:latest
+        prepare: echo libtool autoconf automake pkg-config nghttp2 libressl | xargs -Ix -n1 echo brew '"x"' > /tmp/Brewfile
+        configure: --enable-warnings --enable-werror --enable-debug --with-openssl=$(brew --prefix libressl) --enable-websockets
+
+    - name: macOS arm64 torture
+      env:
+        container_img: ghcr.io/cirruslabs/macos-ventura-xcode:latest
+        prepare: echo libtool autoconf automake pkg-config nghttp2 openssl | xargs -Ix -n1 echo brew '"x"' > /tmp/Brewfile
+        configure: --enable-warnings --enable-werror --enable-debug --disable-shared --disable-threaded-resolver --with-openssl=$(brew --prefix openssl) --enable-websockets
+
+    - name: macOS arm64 torture-ftp
+      env:
+        container_img: ghcr.io/cirruslabs/macos-ventura-xcode:latest
+        prepare: echo libtool autoconf automake pkg-config nghttp2 openssl | xargs -Ix -n1 echo brew '"x"' > /tmp/Brewfile
+        configure: --enable-warnings --enable-werror --enable-debug --disable-shared --disable-threaded-resolver --with-openssl=$(brew --prefix openssl) --enable-websockets
+
+  prepare_script: |
+    ${pkgs} && brew update && for i in 1 2 3; do brew bundle install --no-lock --file /tmp/Brewfile && break || sleep 1; done && sudo -H python3 -m pip install --upgrade pip && sudo -H python3 -m pip install impacket
+  configure_script: |
+    autoreconf -fi && ./configure ${config}
```

I guess you want something like this:

```suggestion
  prepare_script: |
    ${prepare} && brew update && for i in 1 2 3; do brew bundle install --no-lock --file /tmp/Brewfile && break || sleep 1; done && sudo -H python3 -m pip install --upgrade pip && sudo -H python3 -m pip install impacket
  configure_script: |
    autoreconf -fi && ./configure ${configure}
```